frontend/factory-ops/src/data.js

```diff
@@ -1,38 +1,53 @@
 export const ITEM = [
```

Comment:
Change the name to ITEMS; it is an array, and we have many items stored within it.